packages/ag-charts-community/src/chart/navigator/shapes/rangeSelector.ts

```diff
@@ -44,6 +50,8 @@ export class RangeSelector extends Group {
 
     override computeBBox() {
         const { x, y, width, height } = this;
-        return new BBox(x, y, width, height);
+        const minOff = this.minHandle.width / 2;
+        const maxOff = this.maxHandle.width / 2;
+        return new BBox(x - minOff, y, width + (minOff + maxOff), height);
```

Comment:
The goal in my refactor was to make these scenes internally independent so it was more explicit in the `Navigator` class what was happening, as hiding this stuff in the scenes was causing lots of issues with the zoom.

So I'd prefer if in `Navigator.layoutNodes()` it passed in something like:

```
private layoutNodes(x: number, y: number, width: number, height: number) {
    // ...
    rangeSelector.layout(x - minOffset, y, width + (minOffset - maxOffset), height);
    // ...
}
```

So the `RangeSelector` doesn't need to know about the handles and mask directly, and instead the scenes are coordinated by the parent `Navigator`.

Reply:
Done. Moved the handle offset calculations to `Navigator.layoutNodes()`